CONTRIBUTING.md

```diff
@@ -1,14 +1,21 @@
 # Contributing to the ARM Template BPA
-We welcome community contributions to the Template BPA. Please note that by participating in this project, you agree to abide by the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/)  and terms of the [CLA](#contributor-license-agreement-cla).
+We welcome community contributions to the Template BPA. Please note that by participating in this project, you agree to abide by the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/) and terms of the [CLA](#contributor-license-agreement-cla).
 
 ## Getting Started
-* If you haven't already, you will need [dotnet core sdk 3.1](https://dotnet.microsoft.com/download) (or later) installed locally to build and run this project.
+* If you haven't already, you will need the [.NET 5 SDK](https://dotnet.microsoft.com/download) installed locally to build and run this project.
 * Fork this repo (see [this forking guide](https://guides.github.com/activities/forking/) for more information).
 * Checkout the repo locally with `git clone git@github.com:{your_username}/template-analyzer.git`.
-* Build the .NET solution with `dotnet build`.
+* The .NET solution can be built with the `dotnet build` command.
  
 ## Developing
- 
+
+### Environment
+* [Visual Studio Code](https://code.visualstudio.com/) with the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp) is a great way to get started.
+  * Start VS Code and open the root directory where the code is cloned locally (File->Open Folder...).
+* Run the `build` task (Terminal->Run Task...->build) in VS Code to build the Template BPA.
+* Try analyzing a template: open a template file in VS Code and then run the `.NET Core Launch (console)` launch configuration (Run->Start Debugging).
```

Comment:
My VS Code environment is missing the coreclr Extension. Can you add this as a prereq for using VSCode? How do you find this in the Extension Marketplace?

Reply:
I installed VSCode with only the C# extension and was able to follow the instructions :O